media-api/app/model/SyndicationStatus.scala

```diff
@@ -0,0 +1,32 @@
+package model
+
+import play.api.libs.json._
+
+sealed trait SyndicationStatus {
```

Comment:
This is very verbose, we can probably improve using https://github.com/lloydmeta/enumeratum

Also, happy to leave it as it is for now 😃

Reply:
Hmm, we do this already for `payType` and it didn't feel that nice to me...

Reply:
can't remember why though 🤷‍♂️

Reply:
Fair enough. 

I quite like this: https://github.com/guardian/workflow/blob/ae0185d35181fa613fc2169b42406de1c17ebb11/common-lib/src/main/scala/models/Status.scala

Reply:
d'oh it was its [import in `ElasticSearch.scala`](https://github.com/guardian/grid/blob/master/media-api/app/lib/elasticsearch/ElasticSearch.scala#L8) which can be solved by using the `model` package